src/test/scala/akka/contrib/process/BlockingProcessSpec.scala

```diff
@@ -110,15 +112,20 @@ class Receiver(probe: ActorRef, command: String, stdinInput: immutable.Seq[Strin
       sender() ! process
 
     case BlockingProcess.Started(stdin, stdout, stderr) =>
-      FlowGraph.closed(Sink.foreach(probe.!)) { implicit b =>
-        resultSink =>
-          val merge = b.add(Merge[AnyRef](inputPorts = 2))
-          Source(stdout).map(element => Out(element.utf8String)) ~> merge.in(0)
-          Source(stderr).map(element => Err(element.utf8String)) ~> merge.in(1)
-          merge ~> resultSink
-      }
-        .run()
+      RunnableGraph.fromGraph(
+        FlowGraph.create(Sink.foreach[AnyRef](probe.tell(_, Actor.noSender))) { implicit b =>
+          resultSink =>
+            val merge = b.add(Merge[AnyRef](inputPorts = 2))
+            Source(stdout).map(element => Out(element.utf8String)) ~> merge.in(0)
+            Source(stderr).map(element => Err(element.utf8String)) ~> merge.in(1)
+            merge ~> resultSink
+            ClosedShape
+        }
+
+      )
+        .run(materializer)
```

Comment:
I'd recommend avoiding the FlowGraph DSL unless required, which means something like:
~~~scala
al sOut = Source(stdout).map(element => Out(element.utf8String))
val sErr =  Source(stderr).map(element => Err(element.utf8String))
sOut merge sErr runWith Sink.foreach[AnyRef](probe.tell(_, Actor.noSender)) onComplete { _ => self ! "flow-complete" }
~~~